package.json

```diff
@@ -23,19 +23,27 @@
   },
   "homepage": "https://github.com/netlify/netlify-plugin-nextjs#readme",
   "dependencies": {
-    "@netlify/framework-info": "^0.2.0",
     "cpx": "^1.5.0",
     "make-dir": "^3.1.0",
+    "next": "^10.0.1",
```

Comment:
What if the site is using a different version of next, e.g. `9.0.0`?
In that case, this plugin would assume the site is using a different version.

Reply:
oof true

Reply:
this version basically just needs to be aligned with next-on-netlify, which hardcodes its version as  `"next": "9.x"` 

in the future NoN will likely just live in here and so we won't have to manage in multiple places